src/boundaryCondition/DomainBoundary.cpp

```diff
@@ -168,8 +179,8 @@ void DomainBoundary::applyPeriodic(real *dataField, size_t *d_patch, Patch patch
     size_t Nx = domain->get_Nx(level);
     size_t Ny = domain->get_Ny(level);
 
-    int referenceIndex = 0;
-    switch (patch){
+    size_t referenceIndex = 0;
```

Comment:
that is wrong?

Reply:
absolutely right. I overlooked the minus, since get_N{x,y,z} are size_t. Fixed